rocket.go

```diff
@@ -28,6 +31,16 @@ func (rk *Rocket) Mount(route string, h *handler, hs ...*handler) *Rocket {
 	return rk
 }
 
+func (rk *Rocket) Attach(f interface{}) *Rocket {
+	switch v := f.(type) {
+	case *fairing.Response:
+		rk.responseHook = v
+	default:
+		panic("not support fairing")
```

Comment:
web service should not crash(at least not caused by framework), I think better to do normal response? If you don't want others type, better to use **if check** I guess.

Reply:
Yes, here is the initial part of lifecycle.

Reply:
I don't like *

Reply:
Better to keep simple

Reply:
I think `fairing.Response(func())` more intuition than `&fairing.Response{func()}`

Reply:
At?

Reply:
at `Attach()`, more simple to use.

Reply:
We use OnResponse there, and use interface{} do not avoid what you talk about.

Reply:
why not?
user use `Attach(fairing.OnResponse(func()...))` to attach fairing
not `Attach(&fairing.Request{func()...})`
user can easy to use Attach, and we hide the dirty part.
even user use `Attach(&fairing.Request{func()...})`, we still need the interface{} at here.
because we wouldn't have only one fairing.